### ISSUE TITLE
IOS config cleared, when PCMCIA size changed

### ISSUE THREAD

**Issue by ehlers** [GNS3/gns3-server#188]
GNS3: V1.3.3rc1, OS: Mac OS X 10.9.5 and Debian linux 8 (jessie)

When I drop a dynamips router (3725 in my case) into a topology, the config directory looks good:
```
/Users/behlers/GNS3/projects/untitled/project-files/dynamips/configs# ls -l
total 16
-rw-r--r--  1 behlers  staff  1104 14 Mai 13:08 i1_private-config.cfg
-rw-r--r--  1 behlers  staff   402 14 Mai 13:08 i1_startup-config.cfg
```

Then I open the node configurator and change the PCMCIA disk 0 size. The configuration gets cleared by this:
```
/Users/behlers/GNS3/projects/untitled/project-files/dynamips/configs# ls -l
total 0
-rw-r--r--  1 behlers  staff  0 14 Mai 13:09 i1_private-config.cfg
-rw-r--r--  1 behlers  staff  0 14 Mai 13:09 i1_startup-config.cfg
```

When I then start the router, the flash has the correct size, but as the router has no startup and private configuration, the initial configuration dialog starts.

**Comment by grossmj** [comment]
I don't think we should allow users to change the disk sizes after they have dropped their Dynamips. This should be allowed only in IOS router templates in the preferences.

**Comment by ehlers** [comment]
No problem with that. Maybe you can change the node configurator to disallow the disk size change.

**Comment by grossmj** [comment]
I think this is what I will do. Also, changing the disk size for a c7200 leads to a crash and an unusable router...

**Comment by grossmj** [comment]
@ehlers it seems the original issue doesn't happen anymore with the c3725, changing disk sizes seem to work. Do you confirm?

Thanks,

**Comment by ehlers** [comment]
Confirmed, just tested with v1.3.3 on Mac OS X, no problems. Must be resolved between 1.3.3rc1 and 1.3.3, maybe dd9f62158fbd2a8deca149380d12b5268e39a4fa

**Comment by grossmj** [comment]
I love it when bugs get resolved without trying ;)
